### DISCUSSION TITLE
如何将本地的一个xmake项目安装成一个package,有没有一个简单的例子？

### DISCUSSION THREAD

**Discussion by keosu** [xmake-io/xmake#5018] [Q&A] [answered]
Hi，对于远程仓库中已有的依赖包，我们可以通过add_requires, add_package引用；
有没有一个简单项目例子，本地有一个xmake动态库项目,配置package，xmake install, 然后xrepo scan能看到这个package信息，能够被其他项目通过add_package引用

**Comment by star-hengxing** [comment, marked as answer]
https://xmake.io/#/manual/global_interfaces?id=add_repositories
